website/config/fastboot.js

```diff
@@ -0,0 +1,23 @@
+module.exports = function () {
```

Comment:
Needed for the ember-shiki to work in fastboot. see: https://ember-shiki.pages.dev/#fastboot

Reply:
can you add it as a comment in the code itself? thanks